src/couch_views/src/couch_views_indexer.erl

```diff
@@ -541,3 +556,25 @@ key_size_limit() ->
 
 value_size_limit() ->
     config:get_integer("couch_views", "value_size_limit", ?VALUE_SIZE_LIMIT).
+
+
+update_tasks(NumChanges, false) ->
```

Comment:
Can you rename this to `update_active_tasks`

Reply:
removed, no longer needed